module/zfs/vdev.c

```diff
@@ -6580,3 +6580,7 @@ ZFS_MODULE_PARAM_CALL(zfs_vdev, zfs_vdev_, max_auto_ashift,
 	param_set_max_auto_ashift, param_get_uint, ZMOD_RW,
 	"Maximum ashift used when optimizing for logical -> physical sector "
 	"size on new top-level vdevs");
+
+ZFS_MODULE_PARAM_CALL(zfs_vdev, zfs_vdev_, raidz_impl,
+		param_set_raidz_impl, param_get_charp, ZMOD_RW,
```

Comment:
It seems Linux get method is effectively missing.  The `param_get_charp()` will just return "todo" from `zfs_vdev_raidz_impl`, that is never set.  As I can see, `vdev_raidz_impl_get()` is not called on Linux.